changelog/fragments/ansible-path-issue-fix.yaml

```diff
@@ -0,0 +1,9 @@
+entries:
+  - description: >
+      For Ansible operators, API endpoint has path component in it. If it has path then current flow is ignoring the path component.
+    kind: bugfix
+    breaking: true
+    migration:
+      header: Wait for the proper fix.
+      body: >
+        The APImachinary has change related to API endpoint Path fix. Once it got merged then proper fix can be done for this issue. Work on this issue is being tracked here: https://github.com/operator-framework/operator-sdk/issues/4925
```

Comment:
```suggestion
        There is currently no workaround other than running the operator as a pod in the cluster (where it will use the internal endpoint). The fix for this issue is currently blocked waiting on a fix to the apimachinery package. Once the upstream issue is merged then a proper fix will be done for ansible-operator. Work on this issue is being tracked here: https://github.com/operator-framework/operator-sdk/issues/4925
```

Reply:
Modified